app/views/businesses/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<p>
+  <strong>Name:</strong>
+  <%= @business.name %>
+</p>
+
+<p>
+  <strong>Link:</strong>
+  <%= @business.link %>
+</p>
+
+<p>
+  <strong>Description:</strong>
+  <%= @business.description %>
+</p>
```

Comment:
Temp code for internal team. I'll change this to use the same stylings we use for projects in a separate PR